server/xpub-model/entities/file/index.js

```diff
@@ -51,15 +51,23 @@ class File extends BaseModel {
   }
 
   static async find(id) {
-    const [file] = await this.query().where({
-      manuscript_id: id,
+    const file = await this.query().where({
+      id,
```

Comment:
I think `pubsweet-model` has a `find` method on the base class which has this same behaviour, so we may not need to override it anymore.